libs/upd/pipes/src/lib/pipes.module.ts

```diff
@@ -35,10 +35,27 @@ export class LocalePercentPipe implements PipeTransform {
   }
 }
 
+@Pipe({ name: 'translateArray', pure: true })
+export class TranslateArrayPipe implements PipeTransform {
+  constructor(private i18n: I18nService) {}
+
+  transform(values: string[]): string[] {
+    return values.map((value) => this.i18n.translate(value, this.i18n.currentLang));
```

Comment:
Have you tested whether this still works after a language change? I think I remember having issues with that when `pure` was `true`.

Reply:
I didn't have any problems; but if you had issues, I'll change them to false! Also, couldn't get the i18n array working..

Reply:
Ok that's good, as long as it works then it's better to keep it `true`. And for the array translation, it's not a huge deal so it's fine for now.